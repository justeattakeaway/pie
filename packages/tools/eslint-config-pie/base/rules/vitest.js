const vitest = require('eslint-plugin-vitest');

module.exports = {
    plugins: [
        'vitest',
    ],
    rules: {
        ...vitest.configs.recommended.rules,
    },
    globals: {
        ...vitest.environments.env.globals,
    },
};
