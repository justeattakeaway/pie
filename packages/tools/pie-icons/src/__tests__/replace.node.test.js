
// @vitest-environment node
import { test, expect } from 'vitest';
import replace from '../replace';

test('throws an error when run in node environment', () => {
    expect(replace).toThrowErrorMatchingSnapshot();
});
