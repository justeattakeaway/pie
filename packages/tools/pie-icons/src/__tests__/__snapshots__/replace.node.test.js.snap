// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`throws an error when run in node environment 1`] = `"\`pie-icons.replace()\` only works in a browser environment."`;
